### ISSUE TITLE
AttributeError: 'Atom' object has no attribute 'element'

### ISSUE THREAD

**Issue by arjunNUFC** [openmm/openmmforcefields#233]
Maybe this is related to issue #225, but I am getting an error using 
openmmforcefields         0.11.1             pyhd8ed1ab_0    conda-forge 
along with the new OpenFF toolkit 0.11.0
openff-toolkit            0.11.0             pyhd8ed1ab_1    conda-forge

I am using SMIRNOFFTemplateGenerator with openff-2.0.0 as the ligand force field.
And i run into this error:
```
..."/python3.10/site-packages/openmmforcefields/generators/template_generators.py", line 966, in convert_system_to_ffxml
    element=particle.element.symbol, mass=as_attrib(particle.element.mass))
AttributeError: 'Atom' object has no attribute 'element'
```

Is the current version of openmmforcefields not compatible with the new toolkit ?

Thanks all

**Comment by mattwthompson** [comment]
> Is the current version of openmmforcefields not compatible with the new toolkit ?

Correct, we're still working on getting the fix rolled out, sorry.

**Comment by arjunNUFC** [comment]
Ok thanks Matt

**Comment by mattwthompson** [comment]
After #227 this is now merged into the `main` branch. We should wait to close this until a release is made, though.

**Comment by mikemhenry** [comment]
I'm sorry I lied in the release notes :see_no_evil: 
https://github.com/openmm/openmmforcefields/releases/tag/0.11.1

I will get a new release out ASAP.

**Comment by mikemhenry** [comment]
Once we get https://github.com/openmm/openmmforcefields/pull/237 resolved, we should be ready for a release.

**Comment by mikemhenry** [comment]
@arjunNUFC This is now fixed on main, we will get the conda-forge release out soon, but if you install it like this `python -m pip install git+https://github.com/openmm/openmmforcefields.git` then things should work with the new toolkit!

**Comment by arjunNUFC** [comment]
Thanks very much Matt and Mike!

**Comment by mikemhenry** [comment]
Closed now that we have 0.11.2 released :smile:
